src/App/App.test.js

```diff
@@ -1,98 +1,64 @@
 import React from 'react';
 import { fireEvent, screen, render, waitFor } from '@testing-library/react';
 import '@testing-library/jest-dom';
-import { fetchMovies } from '../apiCalls.js'
+import { fetchMovies, fetchSingleMovie, fetchTrailers } from '../apiCalls.js'
 import App from './App';
-import { MemoryRouter, Router } from 'react-router-dom'
-import { createMemoryHistory } from 'history';
+import { MemoryRouter } from 'react-router-dom'
```

Comment:
I'm glad we went with MemoryRouter, as opposed to Router/create memory history, and understand it better now.